documents/Libraries/stdlib/sx-glsl/mx_srgb_to_linear_color3.glsl

```diff
@@ -0,0 +1,13 @@
+void mx_srgb_to_linear_color3(vec3 _in, out vec3 result)
+{
+    vec4 outColor = vec4(_in, 0.);
+    vec4 breakPnt = vec4(0.03928571566939354, 0.03928571566939354, 0.03928571566939354, 1.);
+    vec4 slope = vec4(0.07738015800714493, 0.07738015800714493, 0.07738015800714493, 1.);
+    vec4 scale = vec4(0.9478672742843628, 0.9478672742843628, 0.9478672742843628, 0.9999989867210388);
+    vec4 offset = vec4(0.05213269963860512, 0.05213269963860512, 0.05213269963860512, 9.999989742937032e-07);
+    vec4 gamma = vec4(2.400000095367432, 2.400000095367432, 2.400000095367432, 1.000000953674316);
```

Comment:
There are a couple of more 2.4 gammas (here and below) that should be fixed up. Any word on what the other numbers should be? 

Also some numbers should probably be 1.0 vs 0.9999, and 0 instead of 9-e7. 
0.07... is roughly 1/12.92, 
0.947... is roughly 1.055 / 1.0.  
Not sure about 0.52 as the offset (vs 0.55).